src/db.rs

```diff
@@ -918,9 +918,45 @@ impl Db {
 		Self::open_inner(options, OpeningMode::ReadOnly)
 	}
 
+	/// Drop a column from the database, optionally changing its options.
+	/// This shutdown and restart the db in write mode.
+	pub fn drop_column(&mut self, index: u8, new_options: Option<ColumnOptions>) -> Result<()> {
```

Comment:
Btw there is `clear_column` in migration.rs:
https://github.com/paritytech/parity-db/blob/084ef6ac5eaef96dc23223a89ccb43486ace3eba/src/migration.rs#L158

It removes all data without removing the actual column. I think `clear_column` for this function too will be confusing.

Reply:
oh yes, it is basically doing the same thing (I should have use this :facepalm: ), there is also one for moving column, will fix that redundancy (and make that if column is at last index and no new option is provided, we remove the column (as your use case is the last number it will be usefull).